WebApp.E2ETests/Pages/FeedbackPage.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace WebApp.E2ETests.Pages
+{
+    public class FeedbackPage
+    {
+        private readonly IWebDriver _driver;
+        public const string URI = "http://localhost:8080/#/feedbacks";
```

Comment:
should be parametrized int the future with PSW_WEB_FRONTEND_PUBLISH_HOST env var